dhis2_android_maps/src/main/java/org/dhis2/maps/extensions/FeatureExtensions.kt

```diff
@@ -39,3 +40,11 @@ fun Feature.toStringProperty(): String? = when (source()) {
 
     null -> null
 }
+
+fun Feature.withPlacesProperties(selected: Boolean = false, title: String? = null, subtitle: String? = null) = this.apply {
+    addStringProperty("id", UUID.randomUUID().toString())
+    addBooleanProperty("places", true)
+    addBooleanProperty("selected", selected)
```

Comment:
What do you thing about use constants instead of plain text for this properties?